packages/assets-controllers/src/NftController.ts

```diff
@@ -314,7 +323,7 @@ export class NftController extends BaseController<
    */
   constructor({
     chainId: initialChainId,
-    selectedAddress = '',
+    selectedAccountId = '',
```

Comment:
Similar comment as [here](https://github.com/MetaMask/core/pull/4244#discussion_r1626310421) — what are your thoughts on using the messenger to get this value from the AccountsController a bit later on in this constructor instead of having the constructor take this argument?